stable/prometheus/README.md

```diff
@@ -44,6 +44,7 @@ The following tables lists the configurable parameters of the Prometheus chart a
 
 | Parameter | Description | Default |
 | --------- | ----------- | ------- |
+| `alertmanager.extraArgs` | Additional Alertmanager container arguments | `[]` |
```

Comment:
Change the `[]` to `{}` to make it clear that extraArgs should be a hash/map, not an array/list.

Would be awesome if you did it for `server.extraArgs` as well.